tests/MeAjudaAi.ApiService.Tests/README.md

```
@@ -0,0 +1,66 @@
+# ApiService Tests
+
+## Overview
+
+Testes unitários e de integração para o bootstrapper/web host da aplicação (`MeAjudaAi.ApiService`).
+
+## Project Structure
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)